http-client/src/test/groovy/io/micronaut/http/client/StreamRequestSpec.groovy

```diff
@@ -233,8 +233,8 @@ class StreamRequestSpec extends Specification {
         )), Book).toList().blockingGet()
 
         then:
-        def e= thrown(RuntimeException) // TODO: this should be HttpClientException
-        e != null
+        noExceptionThrown()
```

Comment:
This was throwing already the incorrect exception. It was not throwing a HttpClientException. Not it does not throw an exception at all.